packages/sources/tradingeconomics-test/src/endpoint/stock.ts

```diff
@@ -0,0 +1,50 @@
+import { HttpTransport } from '@chainlink/external-adapter-framework/transports'
+import { ProviderResponseBody } from './price'
+import { StockEndpointTypes } from './stock-router'
+
+type HttpEndpointTypes = StockEndpointTypes & {
+  Provider: {
+    RequestBody: never
+    ResponseBody: ProviderResponseBody[]
+  }
+}
+export const httpTransport = new HttpTransport<HttpEndpointTypes>({
+  prepareRequests: (params, config) => {
+    return params.map((param) => {
+      return {
+        params: [param],
+        request: {
+          baseURL: config.API_ENDPOINT,
+          url: `/symbol/${param.base}`,
+          params: {
+            c: `${config.API_CLIENT_KEY}:${config.API_CLIENT_SECRET}`,
+            f: `json`,
+          },
+        },
+      }
+    })
+  },
+  parseResponse: (params, res) => {
+    const data = res.data[0]
+    return params.map((param) => {
+      if (!res.data || !data || data.Last === undefined) {
+        return {
+          params: param,
+          response: {
+            statusCode: 502,
+            errorMessage: `Tradingeconomics provided no data for ${JSON.stringify(param)}`,
+          },
+        }
+      }
+      return {
+        params: param,
+        response: {
+          data: {
+            result: data.Last,
+          },
+          result: data.Last,
+        },
```

Comment:
Same as the crypto/forex endpoint, could you also include `providerIndicatedTimeUnixMs` here if we have the data available?